multimodal-search-tirg/pods/query-text-read.yml

```diff
@@ -0,0 +1,7 @@
+!FilterQL
```

Comment:
FilterQL is a `driver` not an `executor`. You should use a `!BaseExecutor` with `!FilterQL` as `driver`. And call it `filter_text` so that is not missleading (there is no text reading happening here)